### ISSUE TITLE
ReferenceError: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by RafalWilinski** [stoffern/pokemongo-api#76]
```
var _ref = _asyncToGenerator(regeneratorRuntime.mark(function _callee(user, pass, provider) {
                                   ^

ReferenceError: regeneratorRuntime is not defined
    at /Users/Rafal/Node/pokemon-go-bot/node_modules/pokemongo-api/dist/Auth.js:44:36
    at /Users/Rafal/Node/pokemon-go-bot/node_modules/pokemongo-api/dist/Auth.js:95:6
    at Object.<anonymous> (/Users/Rafal/Node/pokemon-go-bot/node_modules/pokemongo-api/dist/Auth.js:183:2)
    at Module._compile (module.js:541:32)
```

Fresh node project, `pokemongo-api` is the only dependency here. Just qequiring `const API = require('pokemongo-api');` throws this error. Am I doing something wrong?

**Comment by RafalWilinski** [comment]
I've just tried https://github.com/gallexme/pokemongo-api-example-project and it works flawlessly.

**Comment by stoffern** [comment]
Formening this. As babel-polyfill should be added to support es5 after compile

**Comment by aidenwallis** [comment]
I want to avoid using Babel, and this gives me this error, this is a tight project for a high profile client I'm working on, anything I can do to get this working this way? 
`var express = require('express'),
	poke = require('pokemongo-api'),
	player = poke.player,
	router = express.Router();

router.get('/', function(req, res){

	player.Login(null, null, 'google');

});

module.exports = router;`

**Comment by stoffern** [comment]
ES5 working
this is fixed here: 264135f7c834e2d632c2f5327a8c2bb60eccc8d1
